tests/exec-tests.sh

```diff
@@ -33,8 +33,8 @@ RUN sed -i 's|</dev/tty||g' /var/lib/githooks/install.sh && \\
     # Change the base template so we can pass in the hook name and accept flags
     sed -i -E 's|echo.*Hook not run inside a git repository.*|CURRENT_GIT_DIR=".git"|' /var/lib/githooks/base-template.sh && \\
```

Comment:
Maybe unrelated, but why is this needed? All tests steps I looked at create a proper git repo to run inside?

Reply:
True, @rycus86 ?: Try to remove it and we will see, Good that we have lots of tests...

Reply:
Seems to ok to removed according to tests... Les replacements -> better.